packages/react-native-audio-api/android/src/main/cpp/audioapi/CMakeLists.txt

```
@@ -39,6 +32,48 @@ set(LINK_LIBRARIES
   oboe::oboe
 )
 
+if(RN_AUDIO_API_WORKLETS_ENABLED)
+  # Import the worklets library (similar to how Reanimated does it)
+  add_library(worklets SHARED IMPORTED)
+  set_target_properties(
+    worklets
+    PROPERTIES
+      IMPORTED_LOCATION
+      "${REACT_NATIVE_WORKLETS_DIR}/build/intermediates/cmake/${BUILD_TYPE}/obj/${ANDROID_ABI}/libworklets.so"
+  )
+  target_include_directories(
+    react-native-audio-api
+    PRIVATE
+    "${COMMON_CPP_DIR}"
+    "${ANDROID_CPP_DIR}"
+    "${INCLUDE_DIR}"
+    "${INCLUDE_DIR}/opus"
+    "${INCLUDE_DIR}/vorbis"
+    "${REACT_NATIVE_DIR}/ReactCommon"
+    "${REACT_NATIVE_DIR}/ReactCommon/jsiexecutor"
+    "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/react/turbomodule"
+    "${REACT_NATIVE_DIR}/ReactCommon/callinvoker"
+    "${REACT_NATIVE_WORKLETS_DIR}/../Common/cpp"
+    "${REACT_NATIVE_WORKLETS_DIR}/src/main/cpp"
+  )
+  list(APPEND LINK_LIBRARIES worklets)
+else()
+  target_include_directories(
+    react-native-audio-api
+    PRIVATE
+    "${COMMON_CPP_DIR}"
+    "${ANDROID_CPP_DIR}"
+    "${INCLUDE_DIR}"
+    "${INCLUDE_DIR}/opus"
+    "${INCLUDE_DIR}/vorbis"
+    "${REACT_NATIVE_DIR}/ReactCommon"
+    "${REACT_NATIVE_DIR}/ReactCommon/jsiexecutor"
+    "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/react/turbomodule"
+    "${REACT_NATIVE_DIR}/ReactCommon/callinvoker"
+  )
+
+endif()
+
```

I think you can reduce code repetition a little bit here
```suggestion
set(COMMON_INCLUDES
  "${COMMON_CPP_DIR}"
  "${ANDROID_CPP_DIR}"
  "${INCLUDE_DIR}"
  "${INCLUDE_DIR}/opus"
  "${INCLUDE_DIR}/vorbis"
  "${REACT_NATIVE_DIR}/ReactCommon"
  "${REACT_NATIVE_DIR}/ReactCommon/jsiexecutor"
  "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni/react/turbomodule"
  "${REACT_NATIVE_DIR}/ReactCommon/callinvoker"
)

if(RN_AUDIO_API_WORKLETS_ENABLED)
  # Import the worklets library (similar to how Reanimated does it)
  add_library(worklets SHARED IMPORTED)
  set_target_properties(
    worklets
    PROPERTIES
      IMPORTED_LOCATION
      "${REACT_NATIVE_WORKLETS_DIR}/build/intermediates/cmake/${BUILD_TYPE}/obj/${ANDROID_ABI}/libworklets.so"
  )

  list(APPEND COMMON_INCLUDES
    "${REACT_NATIVE_WORKLETS_DIR}/../Common/cpp"
    "${REACT_NATIVE_WORKLETS_DIR}/src/main/cpp"
  )

  list(APPEND LINK_LIBRARIES worklets)
endif()

target_include_directories(
  react-native-audio-api
  PRIVATE
  ${COMMON_INCLUDES}
)
```